airflow/sensors/base.py

```diff
@@ -322,9 +321,3 @@ def mode_setter(_, value):
         return cls_type
 
     return decorate(cls)
-
-
-if 'BUILDING_AIRFLOW_DOCS' in os.environ:
```

Comment:
It's is duplicated code. See: https://github.com/apache/airflow/blob/4099108f554130cf3f87ba33b9d6084a74e70231/airflow/utils/decorators.py#L100